AutocompleteClientTests/FW/Logic/Worker/ConstructorIOIntegrationTests.swift

```diff
@@ -875,12 +876,187 @@ class ConstructorIOIntegrationTests: XCTestCase {
             let cioError = response.error as? CIOError
             let responseData = response.data!
             let browseResult = responseData.results[0]
-
+            
             XCTAssertNil(cioError)
             XCTAssertNotNil(browseResult)
             XCTAssertEqual(browseResult.labels["is_sponsored"], true)
             expectation.fulfill()
         })
         self.wait(for: expectation)
     }
+    
+    func testBrowseItems() {
+        let expectation = XCTestExpectation(description: "Request 204")
+        let query = CIOBrowseItemsQuery(ids: itemIds)
+        self.constructor.browseItems(forQuery: query, completionHandler: { response in
+            let cioError = response.error as? CIOError
+            XCTAssertNil(cioError)
+            expectation.fulfill()
```

Comment:
Could we add some additional assertions here to make sure that the items coming back in the response is what we're requesting for?